plugins/discovery-ec2/src/test/java/org/elasticsearch/cloud/aws/AwsEc2ServiceImplTests.java

```diff
@@ -113,7 +115,8 @@ protected void launchAWSConfigurationTest(Settings settings,
                                               int expectedProxyPort,
                                               String expectedProxyUsername,
                                               String expectedProxyPassword,
-                                              String expectedSigner) {
+                                              String expectedSigner,
+                                              int expectedReadTimeout) {
```

Comment:
As per my previous comments, this has to be fixed if we don't define a default timeout.